src/kixi/nhs/data/storage.clj

```diff
@@ -28,13 +28,8 @@
 
 (defn create-new-dataset
```

Comment:
do create-new-dataset|resource and delete-resource want to be in the main ckan repo?